clients/drcachesim/tools/invariant_checker.cpp

```diff
@@ -696,15 +720,17 @@ invariant_checker_t::check_for_pc_discontinuity(
             TESTANY(OFFLINE_FILE_TYPE_FILTERED | OFFLINE_FILE_TYPE_IFILTERED,
                     shard->file_type_) ||
             // Regular fall-through.
```

Comment:
FYI, there's an additional condition that needs to be added here to check PC discontinuity for signals. It was added by the recently submitted PR #5985 (see `shard->last_signal_context_.xfer_int_pc == 0`).

Reply:
Actually the `shard->last_signal_context_.xfer_int_pc == 0` check is not part of the continuity check, so that's not needed here. See other comment about the `pre_signal_flow_continuity` checks though.